types/deepai/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for deepai 1.0
+// Project: https://deepai.org/
+// Definitions by: NicolasNewman <https://github.com/NicolasNewman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

export as namespace deepai;

```
IMO this is a script which supports global script and ships umd module:
https://github.com/deepai-org/deepai-js-client#installation
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library

Reply:
You're right. In their webpack config they have the library type set to UMD.